scripts/install_python_toolchain.sh

```diff
@@ -7,20 +7,23 @@ if [[ -z $VIRTUALENV ]]; then
   VIRTUALENV=virtualenv
 fi
 
-PIP=pip3
+$VIRTUALENV ${PWD}/deps/env
+source ${PWD}/deps/env/bin/activate
```

Comment:
Isn't `${PWD}/foo` the very definition of how a relative path `foo` is resolved? It could make a difference in strings (abs vs. rel), but for sourcing and things like that it looks like unnecessary verbosity. (Maybe there is something I'm not thinking about?)

For the next line `PYTHON=` it makes sense, because you want that variable to be invariant to changes in pwd. I'm probably overthinking it, but I just got curious about this.

Reply:
Yeah, I think you're right.